### DISCUSSION TITLE
LAB 06.01 - Task 3

### DISCUSSION THREAD

**Discussion by mariaprojas** [rramosp/ai4eng.v1#255] [06 - MÉTODOS NO SUPERVISADOS]
Buenas tardes me sale este error, ya probé el código y funciona bien. Ya intenté saliendo y volviendo a ejecutar todo, desconectando el entorno, y nada :(

![image](https://user-images.githubusercontent.com/113949857/235795604-2038f1f8-13b7-48f4-8d5f-4260a348fb78.png)

![image](https://user-images.githubusercontent.com/113949857/235795570-3bd56430-3ec8-448f-9151-de0dd7b31ae0.png)

Este es el link de mi código: https://colab.research.google.com/drive/1TeJ5Bd8VFj3nuoZ24YK2BfICUbRBLuu1?usp=sharing

**Comment by imEag** [comment]
Hola María, El enlace de colab no me permite el acceso, y además no dejas el código en la pregunta, por lo que no puedo revisar así

**Comment by mariaprojas** [comment]
Hola, el profesor me ayudó a resolverlo, muchas gracias 😊

Cordialmente,




María Paula Rojas Ortega
Ingeniería ambiental
Facultad de ingeniería
Universidad de Antioquia

El jue., 11 de mayo de 2023 10:16 a. m., Emmanuel Arizabaleta Garces <
***@***.***> escribió:

> Hola María, El enlace de colab no me permite el acceso, y además no dejas
> el código en la pregunta, por lo que no puedo revisar así
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/255#discussioncomment-5874574>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A3FLZIPRU6JHNCN23F3PKO3XFT7DFANCNFSM6AAAAAAXTT3PWQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

-- 


"La información aquí contenida es para uso exclusivo de la persona o 
entidad de destino. Está estrictamente prohibida su utilización, copia, 
descarga, distribución, modificación y/o reproducción total o parcial, sin 
el permiso expreso de Universidad de Antioquia, pues su contenido puede ser 
de carácter confidencial y/o contener material privilegiado. Si usted 
recibió esta información por error, por favor contacte en forma inmediata a 
quien la envió y borre este material de su computador. Universidad de 
Antioquia no es responsable por la información contenida en esta 
comunicación, el directo responsable es quien la firma o el autor de la 
misma."


-- 
UdeA

**Reply by imEag** [reply]
perfecto!
